l10n_it_pos_fiscalcode/static/src/js/screens.js

```diff
@@ -0,0 +1,52 @@
+odoo.define("l10n_it_pos_fiscalcode.ClientDetailsEdit", function (require) {
+    "use strict";
+
+    const ClientDetailsEdit = require("point_of_sale.ClientDetailsEdit");
+    const Registries = require("point_of_sale.Registries");
+    const {_t} = require("web.core");
+
+    const PosClientDetailsEdit = (ClientDetailsEdit) =>
+        class extends ClientDetailsEdit {
+            _validCodiceFiscale(cfins) {
+                const cf = cfins.toUpperCase();
+                const cfReg = /^[A-Z]{6}\d{2}[A-Z]\d{2}[A-Z]\d{3}[A-Z]$/;
+                if (!cfReg.test(cf)) {
```

Comment:
`_validFiscalCode`, `fcins`, `fc`, `fcReg` ?

Reply:
... and done

Reply:
> `_validFiscalCode`, `fcins`, `fc`, `fcReg` ?

@robyf70 resta ancora `fcins`.
Per il resto LGTM.